rclc_parameter/README.md

```diff
@@ -0,0 +1,301 @@
+    double param_value = 0.15;
+
+    // Add parameter to the server
+    rcl_ret_t rc = rclc_add_parameter(&param_server, parameter_name, RCLC_PARAMETER_DOUBLE);
+
+    // Set parameter value
+    rc = rclc_parameter_set_double(&param_server, parameter_name, param_value);
+
+    // Get parameter value on param_value
+    rc = rclc_parameter_get_double(&param_server, parameter_name, &param_value);
+    ```
+
+Parameters can also be created by external clients if `allow_undeclared_parameters` flag is set.
+They client just needs to set a value on a unexisting parameter, the parameter will be created if the server is not full and the callback allows the operation.
+
+*Max name size is controlled by the compilation time option `RCLC_PARAMETER_MAX_STRING_LENGTH`, default value is 50.*
+
+## Delete a parameter
+Parameters can be deleted both by the parameter server and external clients:
+```c
+rclc_delete_parameter(&param_server, "param2");
+```
+
+Note that for external delete requests the server callback will be executed, allowing the user to reject the operation.
+
+## Parameters description
+
+- Parameter description  
+    Adds a description of a parameter and its constrains, which will be returned on a describe parameters requests:
+    ```c
+    rclc_add_parameter_description(&param_server, "param2", "Second parameter", "Only even numbers");
+    ```
+
+    *The maximum string size is controlled by the compilation time option `RCLC_PARAMETER_MAX_STRING_LENGTH`, default value is 50.*
+
+- Parameter constrains  
+    Informative numeric constrains can be added to int and double parameters, returning this values on describe parameters requests:
+    - from_value: Start value for valid values, inclusive.
+    - to_value: End value for valid values, inclusive.
+    - step: Size of valid steps between the from and to bound.
+
+        ```c
+        int64_t int_from = 0;
+        int64_t int_to = 20;
+        uint64_t int_step = 2;
+        rclc_add_parameter_constraints_integer(&param_server, "param2", int_from, int_to, int_step);
+
+        double double_from = -0.5;
+        double double_to = 0.5;
+        double double_step = 0.01;
+        rclc_add_parameter_constraints_double(&param_server, "param3", double_from, double_to, double_step);
+        ```
+
+    *This constrains will not be applied by the parameter server, leaving values filtering to the user callback.*
+
+- Read only parameters:  
+    The new API offers a read only flag. This flag blocks parameter changes from external clients, but allows changes on the server side:
+    ```c
+    bool read_only = true;
+    rclc_set_parameter_read_only(&param_server, "param3", read_only);
+    ```
+
+## Cleaning up
+
+To destroy an initialized parameter server:
+
+```c
+// Delete parameter server
+rclc_parameter_server_fini(&param_server, &node);
+```
+
+This will delete any automatically created infrastructure on the agent (if possible) and deallocate used memory on the client side.
```

Comment:
Just to be sure: Did you mean client side or server side?

Reply:
The client on this sentence refers to the micro-ROS client.
Parameter server makes more sense on this context: 7fa6049664809f55ea666fef8c2cd78a8601f956